archivy/templates/click_web/command_form.html

```diff
@@ -11,41 +11,25 @@ <h1 class="command-title">{{ command.name|title }}</h1>
       enctype="multipart/form-data">
     {% set command_list = [] %}
     {% for level in levels %}
-        <fieldset name="{{ level.command.name }}">
-            {% if loop.index == 1 %}
-                {# do not print root command as it most likely is juse "cli" #}
-                <legend class="command-header"></legend>
-            {% elif not level.fields %}
-                {# It is just a command group without any options do not create it's own section #}
-                {% do command_list.append(level.command.name) %}
-            {% else %}
-                {% do command_list.append(level.command.name) %}
-                <legend class="command-header">{{ command_list | join('&ensp;') }}</legend>
-                {% do command_list.clear() %}
-            {% endif %}
+		{% for field in level.fields %}
+			<div class="parameter {{ field.param }} {{ field.type }}">
+				<div class="pure-control-group">
+					{% if field.nargs == -1 %}
+						{{ macros.add_variadic_field_input(field) }}
+					{% else %}
+						<label for="{{ field.name }}">
+							{{ field.human_readable_name|capitalize }}
+						</label>
+						{% for arg in range(field.nargs) %}
+							{{ macros.add_field_input(field) }}
+						{% endfor %}

```

Comment:
looks like you use `vim`. Please set tab = 2, for all `*.html` and `*.js`. and don't forget to lint them with `xmllint`

Reply:
Is 2 spaces recommended for html?

Reply:
2 or 4, this one is 8. which is quite huge